### PR TITLE
bpo-33459: Fix "tuple displays" term in Expressions.rst

### PR DESCRIPTION
**Pull request by andresdelfino**
https://bugs.python.org/issue33459

Automerge-Triggered-By: @matrixise

### CODE REVIEW

File: Doc/reference/expressions.rst [hunk 1]
```diff
@@ -1829,7 +1828,8 @@ precedence and have a left-to-right chaining feature as described in the
 | ``x[index]``, ``x[index:index]``,             | Subscription, slicing,              |
 | ``x(arguments...)``, ``x.attribute``          | call, attribute reference           |
 +-----------------------------------------------+-------------------------------------+
-| ``(expressions...)``,                         | Binding or tuple display,           |
+| ``(expressions...)``,                         | Binding or parenthesized            |
+|                                               | expressions,                        |
```
Comment (vadmium) [inline Doc/reference/expressions.rst]: Doesn’t this split the round bracket syntax into a separate paragraph from the other brackets?

Comment (andresdelfino) [reply Doc/reference/expressions.rst]: Yes, you are right!

**Comment by vstinner** [comment]
I removed the " needs backport to 3.6" label, the 3.6 branch no longer accept bugfixes (only security fixes are accepted): https://devguide.python.org/#status-of-python-branches

**Comment by JulienPalard** [comment]
The proposed change is splitting the last line in two distinct lines. But the table gives one line per priority of precedence, so it would indicate they have different precedence, which looks wrong.

I would keep them in the same block.

**Comment by andresdelfino** [comment]
> The proposed change is splitting the last line in two distinct lines. But the table gives one line per priority of precedence, so it would indicate they have different precedence, which looks wrong.
> 
> I would keep them in the same block.

I agree, just updated the PR.

**Review by matrixise — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**